code/modules/events/false_alarm.dm

```diff
@@ -11,11 +11,18 @@
 /datum/round_event/falsealarm/announce()
 	var/list/events_list = list()
 	for(var/datum/round_event_control/E in SSevent.control)
-		if(!E.wizardevent && !E.holidayID) //No holiday cheer allowed during non-holidays. Not even fake holiday cheer.
-			events_list += E //No holiday cheer allowed during non-holidays. Not even fake holiday cheer.
+		if(E.holidayID || E.wizardevent)
+			continue
+		var/datum/round_event/event = E.typepath
+		if (initial(E.typepath) < 0)
+			continue

```

Comment:
that's probably his point, possibly a side effect of back pain meds?
@MrStonedOne you did a goofball.